example/eks/main.auto.tfvars.dist

```diff
@@ -0,0 +1 @@
+ssh_public_key = "content of a public ssh key of your choice (eg: ~/.ssh/id_rsa.pub)"
```

Comment:
Why extension `.dist` ? Just for curiosity

Reply:
It is a standard Linux (Unix?) practice. Usually, `*.dist` files are sample configuration files you need to edit.